### ISSUE TITLE
[Question] Is there a way to get the model definition at runtime

### ISSUE THREAD

**Issue by rseyferth** [xaviergonz/mobx-keystone#66]
This library looks really good! I'm currently building an application using mobx-state-tree, but running into a lot of trouble especially with Typescript. I was looking into switching to mobx-keystone, but I can't get one thing to work:

Is there a way to get the model definition for a ModelClass at runtime? I looked into the source code, but it seems that all the interesting stuff is not available outside the library itself, is that right? 

So, basically what I'd like to be able to do is something like:

```ts
@model('Test')
class Test extends Model({
  name: tProp(types.string)
}) {}

Test.getProperties().forEach((prop, key) => {
   // [key] would be 'name'
   // [prop] would be the tProp instance
})
```

Is there any way to do this? Thanks!

**Comment by xaviergonz** [comment]
Not currently, I'll think of something though

**Comment by xaviergonz** [comment]
Released as v0.27.0, check `getTypeInfo`

**Comment by rseyferth** [comment]
Thanks, that's great!
